hips/tiles/tile.py

```diff
@@ -95,7 +95,7 @@ def read(cls, meta: HipsTileMeta, filename: str = None) -> 'HipsTile':
 
         if meta.file_format == 'fits':
             hdu_list = fits.open(str(path))
-            data = hdu_list[0].data
+            data = hdu_list[0].data.astype('float')
```

Comment:
@cdeil Yes, this is required. Without this, the pixel values are very small for some reason, maybe they get rounded off without the explicit mention of `float`.